NewEvaluationOrig.html

```diff
@@ -0,0 +1,589 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>New Evaluation</title>
+
+    <link rel="stylesheet/css" type="text/css" href="assets/css/normalize.css" />
+    <link rel="stylesheet/less" type="text/css" href="assets/css/NewEvaluation.less">
+    <script src="less.js" type="text/javascript"></script>
+</head>
+
+<body>
+
+    <header class="header">
```

Comment:
i don't see the purpose of this file and the markup in it: it should contain only a div id="app" not the full markup

Reply:
I use it to always compare the original structure of the HTML when I work on the DOM generating functions. It will be deleted soon